site/en/aurora/index.njk

```diff
@@ -0,0 +1,112 @@
+---
+layout: 'layouts/landing.njk'
+title: 'The Aurora Project'
+description: 'A collaboration between Chrome and open-source web frameworks to improve user experience on the web'
+sections:
+  intro:
+    - url: /blog/introducing-aurora
+      title: What is the Aurora Project?
+      description: Aurora is an initiative where Chrome engineers partner with open-source web frameworks to improve user experience across the web, with a special emphasis on performance.
+    - url: /blog/introducing-aurora
+      title: Why is Aurora important?
+      description: Best practices are constantly evolving. The Aurora team helps bake the latest best practices into web frameworks, so framework developers get the benefits with minimal effort. 
+    - url: /blog/introducing-aurora
+      title: What is Aurora focusing on now?
+      description: In 2023, the Aurora team is focusing on improving Core Web Vitals for applications using web frameworks. We are currently partnering with Next.js, Angular, and Nuxt to ship performance features.
+  projects:
+    - url: /blog/framework-tools-font-fallback/ 
+      title: "Framework tools for font fallbacks"
+      description: "Master how to adjust the size of your font fallbacks to prevent Cumulative Layout Shift (CLS), using tools in Next.js, Nuxt, Vite, and Capsize."
+    - url: /blog/angular-image-directive/ 
+      title: "Optimizing Images with the Angular Image Directive"
+      description: "Learn how the Angular image directive can improve Largest Contentful Paint (LCP), now available in versions 14+."
+    - url: /blog/script-component/ 
+      title: "Optimizing third-party script loading in Next.js"
+      description: "Understand the vision behind the Next.js’ Script component, which provides a built-in solution to optimize the loading of third-party scripts."
+  videos:
+    - url: https://www.youtube.com/watch?v=9Yy9BaRr5Fs 
+      title: "Chrome Aurora"
+      description: "[Infobip Shift 2022] Learn how the Chrome Aurora team is working to advance the web framework ecosystem by collaborating with Next.js, Nuxt, and Angular. "
+    - url: https://www.youtube.com/watch?v=zKQRG8l1OOI 
+      title: "Image Optimization Best Practices with Angular"
+      description: "[ng-conf 2022] This session covers the many ways images can slow down your app, as well as the optimizations the Aurora team recommends to avoid these pitfalls."
```

Comment:
```suggestion
      description: "[ng-conf 2022] This session covers the many ways images can slow down your app, and the optimizations the Aurora team recommends to avoid these pitfalls."
```